src/benchmark/executors.py

```diff
@@ -70,9 +63,6 @@ def __init__(self, log):
         client = docker.from_env()
         self.my_container_dict = {cont.name: cont for cont in client.containers.list()}
 
-    def get_path_to_inference_folder(self):
-        return '/tmp/dl-benchmark/src/inference'
```

Comment:
We use hardcoded path for `Docker executor` because `Docker Executor` is launched from remote machine and can have different paths with Docker Container. For example, the repository is in `/home/` on remote machine.
But DockerFile contains explicit [path](https://github.com/itlab-vision/dl-benchmark/blob/4718626afc4b10326271ae7cdafd70fca4a3bb53/docker/OpenVINO_DLDT/Dockerfile#L66) to the repository. Thus, if remote machine returns `os.path.dirname(os.path.dirname(__file__))`, this path is relative to the local path, not to the path inside Docker container.

We understand that this ways isn't good so we have the issue with the same [problem](https://github.com/itlab-vision/dl-benchmark/issues/248).

Thank you for the attention!

Reply:
Reverted